src/NuGet.CommandLine.StagingExtensions/StageManagementResource.cs

```diff
@@ -154,8 +252,14 @@ public StageManagementResource(string stageServiceUri, HttpSource httpSource)
                 {
                     string serverMessage = await response.Content.ReadAsStringAsync();
 
-                    throw new HttpRequestException(
-                        $"Response status code does not indicate success: {response.StatusCode}. Message: {serverMessage}");
+                    var errorMessage = $"Response status code does not indicate success: {response.StatusCode}.";

```

Comment:
Not a requirement from what I see in NuGet.Protocol.Core.Types. Justin?

---
In reply to: [78231509](https://github.com/NuGet/NuGet.Staging/pull/73#discussion_r78231509) [](ancestors = 78231509)